addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/workGenerators/WorkerGenerator.java

```diff
@@ -310,9 +304,9 @@ public void run() {
                         }
                     } // end of while
                 } catch (FileNotFoundException e) {
-                    e.printStackTrace();
+                    LOG.error("File not found!", e);
```

Comment:
For FileNotFoundExceptions we've found that also outputting the full file path is very helpful - I think the exception often just has the relative path which is not nearly as useful.

Reply:
Thanks, I'll fix all occurrences of FileNotFoundException to log the full file path.